ceph_salt/config_shell.py

```diff
@@ -369,6 +369,18 @@ def possible_values(self):
                 'handler': PillarHandler('ceph-salt:deploy:osd'),
                 'default': False
             },
+            'Bootstrap_Ceph_Conf': {
+                'type': 'group',
+                'help': 'Bootstrap Ceph configuration',
+                'options': {
+                    'Global': {
+                        'type': 'dict',
```

Comment:
can we change this to a simple multi-line string?

Reply:
I've started by using the already existing `list` type, but then I've created the `dict` type where the option name and value are separated giving us some usability advantages:

1) Option name is displayed on the left, and value on the right (more consistency with other `ceph-salt` options)

2) It's possible to rewrite a setting value by simply setting it again (in `list` type we have to delete the old entry and create the new entry to avoid duplicated option)

![Screenshot from 2020-03-18 11-21-31](https://user-images.githubusercontent.com/14297426/76960054-9ea28400-6912-11ea-9b51-9b44e9e8722d.png)

Are you suggesting to change it from `dict` type to `list` type because you are anticipating any issue with `dict` or any advantage in using `list`?

Reply:
I'm just suggesting that we shouldn't be to clever here. I just want to avoid a maintenance burden, if someone wants us to add support for the `[mon]` section. And then later someone wants us to add support for the `[osd]` section, and then later someone wants us to add support for the `[client]` section and then someone wants us to add support for the `[mon.a]` section. And the someone wants us to add support for the `[client.x.y]` section. And then someone wants us to add support for the `[cephfs-shell]` section. And so one.

Reply:
my idea, if users can simply add a multiline string

```
[global]
...
[mon]
....
[mon.a]
...
[client.x]
...
```
we're done.

Reply:
Now I understand what you mean by `multiline string`, I was confusing it with `list of strings`.

I wonder if it's possible to implement `multiline string` input in configshell.

Reply:
I think we can support all types of `sections` by allowing the user to create sections, e.g.:

```
ceph-salt /Deployment/Bootstrap_Ceph_Conf add global
ceph-salt /Deployment/Bootstrap_Ceph_Conf/global set "<key> = <val>"
```